### PR TITLE
Updated Spotlight Blogs list

### PR DESCRIPTION
**Pull request by Devaansh-Kumar**
This pull request updates the list of Spotlight Blogs

### CODE REVIEW

File: communication/contributor-comms/blogging-resources/sig-spotlights.md [hunk 1]
```diff
@@ -14,4 +14,7 @@ These are all the spot lights that are published to date:
...
+| [Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md) | [architecture](https://github.com/kubernetes/kubernetes/labels/sig%2Farchitecture) | [SIG Architecture: Conformance Spotlight](https://kubernetes.io/blog/2023/10/05/sig-architecture-conformance-spotlight-2023/) | Frederico Muñoz | 2023-10-05 |
 | [Testing](https://github.com/kubernetes/community/tree/master/sig-testing#readme) | [testing](https://github.com/kubernetes/kubernetes/labels/sig%2Ftesting) | [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/) | [Sandipan Panda](https://github.com/sandipanpanda) | 2023-11-24 |
+| [Release](https://github.com/kubernetes/community/tree/master/sig-release#readme) | [release](https://github.com/kubernetes/kubernetes/labels/sig%2Frelease) | [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/) | Nitish Kumar | 2024-01-15 |
```
Comment (SD-13) [inline communication/contributor-comms/blogging-resources/sig-spotlights.md:20]: Update this with the spotlight on SIG Release https://kubernetes.io/blog/2024/01/15/sig-release-spotlight-2023/

**Review by SD-13 — CHANGES_REQUESTED** [review]

File: communication/contributor-comms/blogging-resources/sig-spotlights.md (hunk 1 above)
Comment (Devaansh-Kumar) [reply communication/contributor-comms/blogging-resources/sig-spotlights.md]: Yes I have made the changes

File: communication/contributor-comms/blogging-resources/sig-spotlights.md [hunk 2]
```diff
@@ -14,4 +14,7 @@ These are all the spot lights that are published to date:
...
+| [Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md) | [architecture](https://github.com/kubernetes/kubernetes/labels/sig%2Farchitecture) | [SIG Architecture: Conformance Spotlight](https://kubernetes.io/blog/2023/10/05/sig-architecture-conformance-spotlight-2023/) | Frederico Muñoz | 2023-10-05 |
 | [Testing](https://github.com/kubernetes/community/tree/master/sig-testing#readme) | [testing](https://github.com/kubernetes/kubernetes/labels/sig%2Ftesting) | [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/) | [Sandipan Panda](https://github.com/sandipanpanda) | 2023-11-24 |
+| [Release](https://github.com/kubernetes/community/tree/master/sig-release#readme) | [release](https://github.com/kubernetes/kubernetes/labels/sig%2Frelease) | [Spotlight on SIG Testing](https://kubernetes.io/blog/2024/01/15/sig-release-spotlight-2023/) | Nitish Kumar | 2024-01-15 |
```
Comment (SD-13) [inline communication/contributor-comms/blogging-resources/sig-spotlights.md:20]:
```suggestion
| [Release](https://github.com/kubernetes/community/tree/master/sig-release#readme) | [release](https://github.com/kubernetes/kubernetes/labels/sig%2Frelease) | [Spotlight on SIG Release](https://kubernetes.io/blog/2024/01/15/sig-release-spotlight-2023/) | Nitish Kumar | 2024-01-15 |
```

**Review by SD-13 — CHANGES_REQUESTED** [review]

File: communication/contributor-comms/blogging-resources/sig-spotlights.md (hunk 2 above)
Comment (Devaansh-Kumar) [reply communication/contributor-comms/blogging-resources/sig-spotlights.md]: Oh sorry for that :sweat_smile:, I have updated it again

Comment (SD-13) [reply communication/contributor-comms/blogging-resources/sig-spotlights.md]: np

**Pull request merged**